plugins/modules/system/sap_task_list_execute.py

```diff
@@ -0,0 +1,373 @@
+                    elif list_return.text:
+                        text = list_return.text.strip()
+                        if text:
+                            list_out.append(text)
+                # put the list in dictionary;
+                # the key is the tag name the list elements all share in common
+                # the value is the list itself
+                aDict = {element[0].tag: list_out}
+            # if the tag has attributes, add those to the dict
+            if element.items():
+                aDict.update(dict(element.items()))
+            dict_out.update({element.tag: aDict})
+        elif element.items():
+            dict_out.update({element.tag: dict(element.items())})
+        # if there are no child tags and no attributes, extract the text
+        else:
+            dict_out.update({element.tag: element.text})
+    return dict_out
+
+
+def call_rfc_method(connection, method_name, kwargs):
+    # PyRFC call function
+    return connection.call(method_name, **kwargs)
+
+
+def process_exec_settings(task_settings):
+    # processes task settings to objects
+    exec_settings = {}
+    for settings in task_settings:
+        temp_dict = {settings.upper(): 'X'}
+        for key, value in temp_dict.items():
+            exec_settings[key] = value
+    return exec_settings
+
+
+def run_module():
+
+    params_spec = dict(
+        TASKNAME=dict(type='str', required=True),
+        FIELDNAME=dict(type='str'),
+        VALUE=dict(type='raw'),
+    )
+
+    # define available arguments/parameters a user can pass to the module
+    module = AnsibleModule(
+        argument_spec=dict(
+            # values for connection
+            conn_username=dict(type='str', required=True),
+            conn_password=dict(type='str', required=True, no_log=True),
+            host=dict(type='str', required=True),
+            sysnr=dict(type='str', default="00"),
+            client=dict(type='str', default="000"),
+            # values for execution tasks
+            task_to_execute=dict(type='str', required=True),
+            task_parameters=dict(type='list', elements='dict', options=params_spec),
+            task_settings=dict(type='list', elements='str', default=['BATCH']),
+            task_skip=dict(type='bool', default=False),
+        ),
+        supports_check_mode=False,
+    )
+    result = dict(changed=False, msg='', out={}, json_out={}, error={}, )
+
+    params = module.params
+
+    username = (params['conn_username']).upper()
+    password = params['conn_password']
+    host = params['host']
+    sysnr = params['sysnr']
+    client = params['client']
+
+    task_parameters = params['task_parameters']
+    task_to_execute = params['task_to_execute']
+    task_settings = params['task_settings']
+    task_skip = params['task_skip']
+
+    if not HAS_PYRFC_LIBRARY:
+        module.fail_json(
+            msg=missing_required_lib('pyrfc'),
+            exception=ANOTHER_LIBRARY_IMPORT_ERROR)
```

Comment:
Conversely, we should name it:
```suggestion
            exception=PYRFC_LIBRARY_IMPORT_ERROR)
```
;-)